org.gnome.FeedReader.json

```
@@ -1,89 +1,84 @@
+  "runtime-version": "3.30",
+  "sdk": "org.gnome.Sdk",
+  "command": "feedreader",
+  "finish-args": [
+    /* Play sounds */
+    "--socket=pulseaudio",
+    /* Access to X11 and XShm (video playback) */
+    "--socket=x11",
+    "--share=ipc",
+    /* Access to Wayland */
+    "--socket=wayland",
+    /* Access to OpenGL */
+    "--device=dri",
+    /* Access to Network */
+    "--share=network",
+    /* Needed for dconf to work */
+    "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+    "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+    "--talk-name=org.gnome.OnlineAccounts",
+    "--own-name=org.gnome.FeedReader.ArticleView",
+    "--talk-name=org.freedesktop.Notifications",
+    "--talk-name=org.freedesktop.secrets"
+  ],
+  "modules": [{
+      "name": "libgee",
+      "cleanup": ["/include", "*.la", "/lib/pkgconfig", "/share"],
+      "config-opts": ["--enable-introspection=no", "--enable-vala"],
+      "make-install-args": ["girdir=/app/share/gir-1.0", "typelibdir=/app/lib/girepository-1.0"],
+      "sources": [{
+        "type": "archive",
+        "url": "https://download.gnome.org/sources/libgee/0.20/libgee-0.20.1.tar.xz",
+        "sha256": "bb2802d29a518e8c6d2992884691f06ccfcc25792a5686178575c7111fea4630"
+      }]
+    }, {
+      "name": "rest",
+      "config-opts": ["--disable-introspection", "--disable-gtk-doc", "--disable-gtk-doc-html"],
+      "cleanup": ["/include", "*.la", "/lib/pkgconfig", "/share"],
+      "sources": [{
+        "type": "archive",
+        "url": "https://download.gnome.org/sources/rest/0.8/rest-0.8.1.tar.xz",
+        "sha256": "0513aad38e5d3cedd4ae3c551634e3be1b9baaa79775e53b2dba9456f15b01c9"
+      }]
+    }, {
+      "name": "libpeas",
+      "config-opts": ["--disable-gtk-doc", "--disable-gtk-doc-html", "--disable-gtk"],
+      "cleanup": ["/include", "*.la", "/lib/pkgconfig"],
+      "sources": [{
+        "type": "archive",
+        "url": "https://download.gnome.org/sources/libpeas/1.22/libpeas-1.22.0.tar.xz",
+        "sha256": "5b2fc0f53962b25bca131a5ec0139e6fef8e254481b6e777975f7a1d2702a962"
+      }]
+    },
+    {
+      "name": "gnome-online-accounts",
+      "config-opts": ["--disable-telepathy", "--disable-documentation", "--disable-backend"],
+      "sources": [{
+        "type": "archive",
+        "url": "https://download.gnome.org/sources/gnome-online-accounts/3.30/gnome-online-accounts-3.30.0.tar.xz",
+        "sha256": "27d9d88942aa02a1f8d003dfe515483d8483f216ba1e297a8ef67a42cf4bcfc3"
+      }]
+    },
+    {
+      "name": "gumbo",
+      "sources": [{
+        "type": "archive",
+        "url": "https://github.com/google/gumbo-parser/archive/v0.10.1.tar.gz",
+        "sha256": "28463053d44a5dfbc4b77bcf49c8cee119338ffa636cc17fc3378421d714efad"
+      }]
+    },
+    {
+      "name": "feedreader",
+      "buildsystem": "meson",
+      "config-opts": ["--buildtype=release"],
+      "sources": [{
+        "type": "archive",
```

```suggestion
        "type": "git",
```
Should point to the git url instead of the release tarball

---

Should be fixed now